.jshintrc

```diff
@@ -9,6 +9,7 @@
   "unused": false,
 
   "globals": {
-    "OpenSeadragon": true
+    "OpenSeadragon": true,
+    "define": false
```

Comment:
So, apparently, if the value is `false` it means that JSHint will consider that gloabl variable as "read only", whereas if it's `true` it means that reading and writing to that global is ok.